recipe/meta.yaml

```diff
@@ -43,6 +43,7 @@ outputs:
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
       host:
         - cuda-version {{ cuda_version }}
+        - expat
```

Comment:
Saw this error [on CI]( https://app.travis-ci.com/github/conda-forge/cuda-gdb-feedstock/jobs/603465413#L819 )

```
['  ERROR (cuda-gdb,bin/cuda-gdb): $RPATH/libexpat.so.1 not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?']
```

This only happens on `aarch64` and `ppc64le`. Though added `expat generally in case it is actually needed for `x86_64` as well